### PR TITLE
enabled dev and stable builds

### PR DESCRIPTION
**Pull request by HarithaIBM**
enabled dev and stable builds

### CODE REVIEW

**Review by MikeFultonDev — APPROVED** [review]
nice! LGTM

File: cicd-dev.groovy [hunk 1]
```diff
@@ -10,6 +10,6 @@ node('linux')
         }
 
         stage('Build') {
-                build job: 'Port-Pipeline', parameters: [string(name: 'PORT_GITHUB_REPO', value: 'https://github.com/ZOSOpenTools/gitport.git'), string(name: 'PORT_DESCRIPTION', value: 'git description' )]
+                build job: 'Port-Pipeline', parameters: [string(name: 'PORT_GITHUB_REPO', value: 'https://github.com/ZOSOpenTools/gitport.git'), string(name: 'PORT_DESCRIPTION', value: 'git description' ), string(name: 'BUILD_LINE', value: 'DEV')]
```
Comment (IgorTodorovskiIBM) [inline cicd-dev.groovy:13]: No related to this PR, but are you able to update the "git description" to "a distributed version control system "

File: cicd-stable.groovy [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+                checkout([
+                        $class: 'GitSCM',
+                        branches: [[name: '*/main']],
+                        doGenerateSubmoduleConfigurations: false,
+                        extensions: [],
+                        userRemoteConfigs: [[url: 'https://github.com/ZOSOpenTools/gitport.git']]])
+        }
+
+        stage('Build') {
+                build job: 'Port-Pipeline', parameters: [string(name: 'PORT_GITHUB_REPO', value: 'https://github.com/ZOSOpenTools/gitport.git'), string(name: 'PORT_DESCRIPTION', value: 'git description' ), string(name: 'BUILD_LINE', value: 'STABLE')]
```
Comment (IgorTodorovskiIBM) [inline cicd-stable.groovy:13]: Same here

**Review by IgorTodorovskiIBM — APPROVED** [review]
LGTM, we can handle the description update in another PR

**Comment by v1gnesh** [comment]
Oooh, this makes some/many of my bump patches obsolete, as some are looking for `TARBALL_URL` (for example) to even update the download URL :( @IgorTodorovskiIBM

File: cicd-dev.groovy (hunk 1 above)
Comment (HarithaIBM) [reply cicd-dev.groovy:13]: Updated in PR : https://github.com/ZOSOpenTools/gitport/pull/89/files

File: cicd-stable.groovy (hunk 2 above)
Comment (HarithaIBM) [reply cicd-stable.groovy:13]: Updated in PR: https://github.com/ZOSOpenTools/gitport/pull/89/files

**Pull request merged**